src/privacy-policy.js

```diff
@@ -0,0 +1,63 @@
+let utils = require('./utils')
+
+class PrivacyPolicy {
+    addLists (lists) {
+        this.tosdrList = lists.tosdr
+        this.polisisList = lists.polisis
+
+        this.addDuplicateRulesForParents(this.tosdrList)
+        this.addDuplicateRulesForParents(this.polisisList)
+    }
```

Comment:
I've created a separate module that matches up privacy policy data with a URL, since we discussed it doesn't belong in the Grade class anymore.

I think we may end up using a very similar class when we port this to the extension.